users/users.md

```diff
@@ -0,0 +1,319 @@
+    "loginName": "template.mona.benson",
+    "email": "mona.benson@sample.com",
+    "accountActive": true,
+    "city": "Sydney",
+    "country": "Australia",
+    "dateOfBirth": null,
+    "department": null,
+    "lastSignInDate": "2011-03-14T14:56:00.0000000",
+    "phoneNumberLand": null,
+    "phoneNumberMobile": null,
+    "timeZoneHasDaylightSavings": true,
+    "timezone": "(UTC+10:00) Canberra, Melbourne, Sydney",
+    "userRole": "Designer"
+}
+```
+---
+
+## Disable user account
+
+Disable a user account.
+
+```apacheconfig
+POST /v1/users/{useId}
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userId | `string` | The user id to request. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+```json
+{
+    "accountActive": false
+}
+```
+
+#### Response body
+The successful response contains an array of instances of a [User](../users/users_schemas.md/#user).
+
+The error response contains an [Error](../response_codes.md).
+
+### Example
+
+Request
+
+```shell
+curl "https:///app.solvexia.com/api/v1/users/u-11427" -X POST -H "Authorization: Bearer syPHeMY5H--kdRtfpoXTgYFF7LHgVOhIjOQ5QkIvSD68VZvc2_uAew.P07tEVThD5SqNCV_tFwbAg" -H "Content-Type: application/json" -d '{"accountActive": false}'
+```
+
+Response
+
+```json
+{
+    "id": "u-11427",
+    "firstName": "Mona",
+    "lastName": "Benson",
+    "loginName": "template.mona.benson",
+    "email": "mona.benson@sample.com",
+    "accountActive": false,
+    "city": "Sydney",
+    "country": "Australia",
+    "dateOfBirth": null,
+    "department": null,
+    "lastSignInDate": "2011-03-14T14:56:00.0000000",
+    "phoneNumberLand": null,
+    "phoneNumberMobile": null,
+    "timeZoneHasDaylightSavings": true,
+    "timezone": "(UTC+10:00) Canberra, Melbourne, Sydney",
+    "userRole": "Designer"
+}
+```
+---
+
+## Get user permission
+
+Get a list of user permissions.
+
+```apacheconfig
+GET /v1/users/{userId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userId | `string` | The user id to request. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+The request body must be empty.
+
+#### Response body
+The successful response contains a list of [Permission](./users_schemas.md/#permission).
+
+The error response contains an [Error](../response_codes.md).
+
+### Example
+
+Request
+
+```shell
+curl "https://app.solvexia.com/api/v1/users/u-11427/permisions" -X GET -H "Authorization: Bearer syPHeMY5H--kdRtfpoXTgYFF7LHgVOhIjOQ5QkIvSD68VZvc2_uAew.P07tEVThD5SqNCV_tFwbAg" 
+```
+
+Response
+
+```json
+[
+  {
+    "resourceId": "p-2343",
+    "resourceName": "Sales reconciliation",
+    "role": "executor"
+  },
+  {
+    "resourceId": "mt-4323",
+    "resourceName": "Monthly Revenue",
+    "role": "reader"
+  }
+]
+```
+---
+
+## Add or update user permission
```

Comment:
Add or update user permission for a given resource

Reply:
Same for the user group

Reply:
updated